README.md

```diff
@@ -25,8 +25,9 @@ Once you have enough fun with the sample app, run the following script to clean
 
 Some other commands that might be helpful:
 ```bash
-./scripts/cf_deploy push    # builds and pushes frontend and backend
-./scripts/cf_deploy rebind  # unbinds and rebinds frontend and backend
+./scripts/cf_deploy push                     # builds and pushes frontend and backend
+./scripts/cf_deploy bound_app_config_update  # update bound apps' configuration with calling the update endpoint on the backing app. You will need to be a space developer to do so.
```

Comment:
I wonder if `dynamic_config_update` might be a friendlier name to users? Although, if user knows CF then this name makes sense too. Not a blocker from my perspective, just a suggestion/thought.

Reply:
Is `dynamic` the word we are going to use in our doc to communicate this feature? If so, I'm happy to use it here. Users may not know which config we are updating given routes can also be configured at creation time, what about `dynamic-binding-update`?

Reply:
I like it! Yes, I think dynamic binding update is a great way to describe this approach via API. Thank you.

Reply:
I would +1 putting some thought into what we call it in general--here and in docs.